src/rustdoc-json-types/lib.rs

```diff
@@ -281,19 +281,28 @@ pub enum StructType {
     Unit,
 }
 
+#[non_exhaustive]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash)]
+#[serde(rename_all = "snake_case")]
+pub enum Modifiers {
```

Comment:
The [reference](https://doc.rust-lang.org/reference/items/functions.html) calls them `Qualifiers`.

Reply:
I like naming for consistency, switched